apps/fluent-tester/src/E2E/Callout/specs/Callout.spec.win.ts

```diff
@@ -19,26 +21,40 @@ describe('Callout Testing Initialization', function () {
   });
 });
 
-/* This will be re-enabled after the Callout's ViewManager integrates the testID prop. This is used to select UI components */
-// describe('Callout Functional Testing', () => {
-//   it('Open the callout and validate it loaded correctly (visible)', () => {
-//     CalloutPageObject.scrollToTestElement();
-//     CalloutPageObject.openCallout();
-//     CalloutPageObject.waitForCalloutComponentInView(PAGE_TIMEOUT);
-//     expect(CalloutPageObject.didCalloutLoad()).toBeTruthy();
-//   });
-// });
-
-// describe('Callout Accessibility Testing', () => {
-//   it('Validate accessibilityRole is correct', () => {
-//     CalloutPageObject.scrollToTestElement();
-//     CalloutPageObject.waitForCalloutComponentInView(PAGE_TIMEOUT);
-//     expect(CalloutPageObject.getAccessibilityRole()).toEqual(CALLOUT_A11Y_ROLE);
-//   });
-
-//   it('Set accessibilityLabel', () => {
-//     CalloutPageObject.scrollToTestElement();
-//     CalloutPageObject.waitForCalloutComponentInView(PAGE_TIMEOUT);
-//     expect(CalloutPageObject.getAccessibilityLabel(CalloutSelector.Primary)).toEqual(CALLOUT_ACCESSIBILITY_LABEL);
-//   });
-// });
+describe('Callout Accessibility Testing', () => {
+  beforeAll(async () => {
+    await CalloutPageObject.scrollToTestElement();
+    await CalloutPageObject.openCallout();
+    await CalloutPageObject.waitForCalloutComponentInView(PAGE_TIMEOUT);
+  });
+
+  it('Validate accessibilityRole is correct', async () => {
+    await expect(await CalloutPageObject.getAccessibilityRole()).toEqual(CALLOUT_A11Y_ROLE);
+    await expect(await CalloutPageObject.didAssertPopup()).toBeFalsy(CalloutPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
+  });
+
+  it('Set accessibilityLabel', async () => {
+    await expect(await CalloutPageObject.getAccessibilityLabel(ComponentSelector.Primary)).toEqual(CALLOUT_ACCESSIBILITY_LABEL);
+    await expect(await CalloutPageObject.didAssertPopup()).toBeFalsy(CalloutPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
+  });
+
+  afterAll(async () => {
+    await CalloutPageObject.closeCallout();
+  });
+});
+
+describe('Callout Functional Testing', () => {
+  beforeEach(async () => {
+    await CalloutPageObject.scrollToTestElement();
+    await CalloutPageObject.openCallout();
+    await CalloutPageObject.waitForCalloutComponentInView(PAGE_TIMEOUT);
+  });
+
+  it('Open the callout and validate it loaded correctly (visible)', async () => {
+    await expect(await CalloutPageObject.didCalloutLoad()).toBeTruthy();
+  });
```

Comment:
While we're at it, maybe let's add some more tests? We could test the `DismissBehaviors` prop

Reply:
I'd also be ok with more tests being its own PR

Reply:
Sadly @rurikoaraki this issue attributes back to our fix for Menu in the rex-win32 tester (vs other host apps); I identified it w/ Larry and we'll take an issue down for later follow up